Makefile

```diff
@@ -151,20 +155,21 @@ codegen-controller:
 		object:headerFile=hack/boilerplate.go.txt \
 		paths=./...
 
-.PHONY: codegen-docs
-codegen-docs:
-	npm install -g @bitnami/readme-generator-for-helm
-	bash hack/helm-docs/helm-docs.sh
-
-.PHONY: codegen-proto
-codegen-proto:
-	./hack/codegen/proto.sh
+.PHONY: codegen-directive-configs
+codegen-directive-configs:
+	npm install -g quicktype
+	./hack/codegen/directive-configs.sh
 
 .PHONY: codegen-ui
 codegen-ui:
 	pnpm --dir=ui install --dev
 	pnpm --dir=ui run generate:schema
 
+.PHONY: codegen-docs
+codegen-docs:
+	npm install -g @bitnami/readme-generator-for-helm
+	bash hack/helm-docs/helm-docs.sh
+
```

Comment:
No changes here. I just re-ordered the targets so they are defined in the same order they are executed when using `make codegen`. I thought it made it slightly easier to follow.